### ISSUE TITLE
[JENKINS-9110] Add Build Date to Junit Test Result (History and Case View)

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/junit-plugin#890]
<p>We like to have a tiny improvement in the junit reports view. Beside the time it took and the build number we need the run date/time as well for our review process. Just the build number is not enough.</p>

<p>See attached patch for the changes.<br/>
Basically it adds in the history view the date for each build next to the Buildnumber and in the individual reports the date beneath the time it took.</p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=lukasfunk">lukasfunk</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-9110" target="_blank">Add Build Date to Junit Test Result (History and Case View)</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=chrisa89">chrisa89</a>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>component(s)</b>: junit-plugin
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 1
<li><b>watchers</b>: 4
<li><b>imported</b>: 2025-11-26
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>We like to have a tiny improvement in the junit reports view. Beside the time it took and the build number we need the run date/time as well for our review process. Just the build number is not enough.</p>

<p>See attached patch for the changes.
Basically it adds in the history view the date for each build next to the Buildnumber and in the individual reports the date beneath the time it took.</p></pre>
</details>
</details>
<details><summary><i>1 attachment</i></summary>

- [patch.diff](https://issues.jenkins.io/secure/attachment/20314/patch.diff)
</details>
